Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -43,7 +43,7 @@ async function main() {
         await appServiceUtility.pingApplication();
         let kuduService: Kudu = await appServiceUtility.getKuduService();
         kuduServiceUtility = new KuduServiceUtility(kuduService);
-        tl.setVariable('AppServiceApplicationUrl', await appServiceUtility.getApplicationURL());
+        tl.setVariable('AppServiceApplicationUrl', await appServiceUtility.getApplicationURL(!taskParams.isLinuxApp ? taskParams.VirtualApplication : null));
```

Comment:
Why this is specific to LinuxApp. What's the behavior for VirtualApp in Windows?

Reply:
This is specific for Windows app. Virtual application is not available in Linux platform.